lib/jxl/render_pipeline/stage_xyb.cc

```diff
@@ -39,18 +40,20 @@ class XYBStage : public RenderPipelineStage {
     msan::UnpoisonMemory(row0 + xsize, sizeof(float) * (xsize_v - xsize));
     msan::UnpoisonMemory(row1 + xsize, sizeof(float) * (xsize_v - xsize));
     msan::UnpoisonMemory(row2 + xsize, sizeof(float) * (xsize_v - xsize));
+    // TODO(eustas): investigate if it is allowed to start 'earlier' for
```

Comment:
You can use up to kPipelinePaddingX (I don't remember the name) bytes before and after the data